lib/mix/task.ex

```diff
@@ -20,4 +24,17 @@ defmodule Antikythera.Mix.Task do
     {:ok, _} = Application.ensure_all_started(Antikythera.Env.antikythera_instance_name())
     :ok
   end
+
+  @doc """
+  Set the specified `node_id` to the `Antikythera.ContextId` in GearLog.
+
+  If you want to use GearLog in mix task, you must set `node_id` before you call GearLog functions.
+
+  The `Antikythera.ContextId` in GearLog will become `{timestamp}_{node_id}_{PID}`.
+  The `timestamp` and `PID` are automatically got from system.
+  """
+  defun set_node_id_to_gear_log_context(node_id :: v[String.t]) :: :ok do
```

Comment:
I'm concerned about the fact that I can't read the node_id constraints just by looking at this definition and documentation.
Can't we define a regular expression that can be specified as follows?
```ex
defmodule NodeId do
  use Croma.SubtypeOfString, pattern: ~R/\A[0-9A-Za-z.-]+\z/
end

defun set_node_id_to_gear_log_context(node_id :: v[NodeId.t]) :: :ok do
```

Reply:
Added `NodeId` in https://github.com/access-company/antikythera/pull/126/commits/cc3283e35cd770d0edd69bf701330da193f4ccd2

`AntikytheraCore.Cluster.NodeId.get/0` should return `~R/\A[0-9A-Za-z.-]+\z/`, but no one explicitly check it. I changed `AntikytheraCore.Cluster.NodeI` to use `Antikythera.NodeId` to verify it.  https://github.com/access-company/antikythera/pull/126/commits/30b24d726f96b681990a4ea6f2956e69951e0ad1